README.md

```diff
@@ -24,6 +24,7 @@
   * Join the #data-archiving slack channel
   * DM a guide your email address for an invite to the pipeline app
   * Please use the same username for both slack and app ("firstnamelastname" or whatever you are comfortable with having online)
+  * Follow the [data archiving instructions](./workflow/harvesting.md)
```

Comment:
Did you want to link specifically to the harvesting workflow here? Seems like people in this track need to follow both/either the research workflow and the harvesting workflow, and the research comes first.